src/main/StreamTabsScreen.js

```diff
@@ -44,35 +49,38 @@ type Props = $ReadOnly<{|
 
 export default function StreamTabsScreen(props: Props) {
   return (
-    <Tab.Navigator
-      {...materialTopTabNavigatorConfig({
-        showLabel: true,
-        showIcon: false,
-      })}
-      swipeEnabled
-    >
-      <Tab.Screen
-        name="subscribed"
-        component={SubscriptionsCard}
-        options={{
-          tabBarLabel: ({ color }) => (
-            <Text style={[styles.tab, { color }]}>
-              <FormattedMessage id="Subscribed" defaultMessage="Subscribed" />
-            </Text>
-          ),
-        }}
-      />
-      <Tab.Screen
-        name="allStreams"
-        component={StreamListCard}
-        options={{
-          tabBarLabel: ({ color }) => (
-            <Text style={[styles.tab, { color }]}>
-              <FormattedMessage id="All streams" defaultMessage="All streams" />
-            </Text>
-          ),
-        }}
-      />
-    </Tab.Navigator>
+    <SafeAreaView style={styles.container}>
```

Comment:
Can you include a screenshot of what this looks like before and after on iOS, so I can see what the effect `SafeViewArea` has on the layout?

Reply:
I can't include the screenshots because I dont have MacBook to compile and run  ( _if nyone looking at it please include screenshots I will be thankful to you_ 

previously the loadingBanner was in both seperate Tabs ie. /streams/SubscriptionsCard and /subscriptions/StreamListCard and both component includes in main/StreamTabsScreen as Tabs, so I removed loadingBanner from both and added single progressBar in parent screen(StreamTabsScreen) so I can show the loading status in the NavigationBar or say parallel to tab's Indicator as shown in image.
<img src="https://user-images.githubusercontent.com/31806059/111976556-c29d3480-8b27-11eb-8018-35f8702398c2.png" />
Again there need a **parent** to hold the tab's parent compontent(Tab.Navigator) and progressBar and for this I could use simple View or SafeAreaView, once look at the [link](https://stackoverflow.com/a/60557715/9929519) it's not got many upvotes but i agreed with the answer. As progressBar is placed out side of Tab.Navigator component so I used SafeAreaView well it doesn't have any effect because of progressBar `style={position:'absolute', top:NAVBAR_SIZE}` it will always lie at bottom of Navbar but to be in safer side I used it, it will act as simple View in android and IOS-version 10 or lower.

Reply:
I don't have any apple devices either, so I can't test it, unfortunately.

But I think it makes more sense to just use a plain `View`, unless there's a specific layout problem that the `SafeAreaView` is solving. Feel free to start a thread in #mobile-team on CZO if you want more people's input on this, though, since I don't have much iOS experience.

This seems especially true given this bit from the [React Navigation docs](https://reactnavigation.org/docs/handling-safe-area/):

> While React Native exports a `SafeAreaView` component, it has some inherent issues, i.e. if a screen containing safe area is animating, it causes jumpy behavior.

Reply:
Lets work harder to get one then :D 
Now I think I should wrap them in View :)